draft-ietf-quic-manageability.md

```diff
@@ -945,37 +945,59 @@ severe packet loss.
 packets, flows, or some other aggregate) into "good" (productive) and "bad"
 (DDoS) traffic, and then differential treatment of this traffic to forward only
 good traffic. This operation is often done in a separate specialized mitigation
 environment through which all traffic is filtered; a generalized architecture
 for separation of concerns in mitigation is given in
- {{?DOTS-ARCH=I-D.ietf-dots-architecture}}.
+{{?DOTS-ARCH=I-D.ietf-dots-architecture}}.
 
-Key to successful DDoS mitigation is efficient classification of this traffic in
-the mitigation environment.  Limited first-packet garbage detection as in
-{{sec-garbage}} and stateful tracking of QUIC traffic as in {{sec-stateful}}
-above may be useful during classification.
+Efficient classification of this DDoS traffic in the mitigation environment
+is key to the success of this approach. Limited first-packet garbage detection
+as in {{sec-garbage}} and stateful tracking of QUIC traffic as in
+{{sec-stateful}} above may be useful during classification.
 
 Note that the use of a connection ID to support connection migration renders
 5-tuple based filtering insufficient and requires more state to be maintained by
-DDoS defense systems. For the common case of NAT rebinding, DDoS defense systems
+DDoS defense systems. For the common case of NAT rebinding, where the client's
+address changes without the client's intent or knowedge, DDoS defense systems
 can detect a change in the client's endpoint address by linking flows based on
-the server's connection IDs. QUIC's linkability resistance ensures that
+the server's connection IDs. However, QUIC's linkability resistance ensures that
 a deliberate connection migration is accompanied by a change in the connection
-ID.
-
-It is questionable whether connection migrations must be supported during a DDoS
-attack. If the connection migration is not visible to the network that performs
-the DDoS detection, an active, migrated QUIC connection may be blocked by such a
-system under attack. As soon as the connection blocking is detected by the
-client, the client may rely on the fast resumption mechanism provided by
-QUIC. When clients migrate to a new path, they should be prepared for the
-migration to fail and attempt to reconnect quickly.
-
-TCP syncookies {{?RFC4937}} are a well-established method of mitigating some
+ID. In this case, the connection ID can not be used to distinguish valid, active
+traffic from new attacl traffic.
+
+Alternatively, when an encrypted transport is used, it is also possible for
+endpoints to directly support these security functions.
+Endpoints can cooperate with an on-path device directly by e.g. sharing
+information about connection IDs.
+
+Another potential method could use an
+on-path network device that relies on pattern inferences in the traffic and
+heuristics or machine learning instead of processing observed header
+information.
+
+However, it is questionable whether connection migrations must be supported
```

Comment:
Is this referring to just explicit connection migration or do you include NAT rebinding as well? Either way, I think it should be explicitly called out. I think, ideally, NAT rebinding should continue to work, but I tend to agree full, explicit connection migration likely isn't completely necessary while under attack.